salt/cache/redis_cache.py

```diff
@@ -203,7 +219,12 @@ def __virtual__():
     if not HAS_REDIS:
         return (False, "Please install the python-redis package.")
     if not HAS_REDIS_CLUSTER and _get_redis_cache_opts()["cluster_mode"]:
-        return (False, "Please install the redis-py-cluster package.")
+        return (
+            False,
+            "Please install the redis python client at least version 4.1.0-rc1 with cluster\n"
+            "support added (https://github.com/redis/redis-py/releases/tag/v4.1.0rc1)."
+        )
```

Comment:
**suggestion**: when testing I discovered that redis-py-cluster is incompatible with redis>=4 - we should probably note that in the module documentation (maybe move this info into the toplevel mod doc and link to the cache documentation instead)